platform/legacy/LegacyPlatformProxifier.ts

```diff
@@ -0,0 +1,90 @@
+import { Server } from 'net';
+import { IncomingMessage, OutgoingMessage } from 'http';
+import { EventEmitter } from 'events';
+
+import { Logger } from '../logging';
+
+/**
+ * Represents "proxy" between legacy and current platform.
+ * @internal
+ */
+export class LegacyPlatformProxifier extends EventEmitter {
+  private server: Server;
+  private readonly eventHandlers: any;
+
+  constructor(
+    private readonly log: Logger,
+    private readonly startPlatform: () => Promise<void>,
+    private readonly stopPlatform: () => Promise<void>
+  ) {
+    super();
+
+    const forwardEvent = (name: string) => (name: string, ...args: any[]) => {
```

Comment:
Remove `name: string, ` from second function here

Reply:
Oh, yeah, good catch. I'm really surprised that it still works :)

Reply:
Ah, main `'request'` event is still being forwarded correctly from another method